### PR TITLE
Update `ISSUE_TEMPLATE` config

### PR DESCRIPTION
**Pull request by alectrocute**
Hi Lemdevs!

It's my opinion that we can greatly improve our `ISSUE_TEMPLATE` config here on this repo.

Please see the comments below for more details.

Thanks!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 1]
```diff
@@ -1,66 +1,67 @@
-name: "\U0001F41E Bug Report"
-description: Create a report to help us improve lemmy-ui
-title: "[Bug]: "
+name: "Bug report"
```
Comment (alectrocute) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:1]: No need for emojis.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 2]
```diff
@@ -1,66 +1,67 @@
...
+description: Create a bug report to help us improve Lemmy-UI!
+title: ""
```
Comment (alectrocute) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:3]: It already has a "bug" label, so no need for the prefix. Just adds more text for us poor humans to read.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 3]
```diff
@@ -1,66 +1,67 @@
...
 labels: ["bug", "triage"]
 body:
   - type: markdown
     attributes:
       value: |
-        Found a bug? Please fill out the sections below. 👍
-        Thanks for taking the time to fill out this bug report!
-        For backend issues, use [lemmy](https://github.com/LemmyNet/lemmy/issues/new/choose)
+        Thanks for taking the time to help improve Lemmy-UI by reporting a bug!
```
Comment (alectrocute) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:9]: Making the backend details one of the steps below, so no need to have additional text here.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 4]
```diff
@@ -1,66 +1,67 @@
...
   - type: checkboxes
     attributes:
       label: Requirements
-      description: Before you create a bug report please do the following.
+      description: Before you create a bug report, please carefully check the following –
       options:
-        - label: Is this a bug report? For questions or discussions use https://lemmy.ml/c/lemmy_support
+        - label: This is a bug report, and if not, I'd post to https://lemmy.ml/c/lemmy_support instead.
```
Comment (alectrocute) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:15]: Making it in the format of "I agree to...", which feels more impactful.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 5]
```diff
@@ -1,66 +1,67 @@
...
       required: true
   - type: textarea
     id: reproduce
     attributes:
       label: Steps to Reproduce
       description: |
-        Describe the steps to reproduce the bug.
-        The better your description is _(go 'here', click 'there'...)_ the fastest you'll get an _(accurate)_ resolution.
+        In a numbered list, walk us through the steps needed to reproduce the bug.
+        The better your description is _(go 'here', click 'there'...)_, the quicker we can fix it.
```
Comment (alectrocute) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:36]: More conversational.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 6]
```diff
@@ -1,66 +1,67 @@
...
       required: true
   - type: input
     id: lemmy-ui-version
     attributes:
-      label: Version
-      description: Which Lemmy UI version do you use? Displayed in the footer.
-      placeholder: ex. 0.17.4-rc.4
+      label: Lemmy Instance Version
+      description: What's the version of the Lemmy instance where the bug can be reproduced?
+      placeholder: ex. 0.18-rc.6
```
Comment (alectrocute) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:59]: More specific.

File: .github/ISSUE_TEMPLATE/QUESTION.yml
Comment (alectrocute) [inline .github/ISSUE_TEMPLATE/QUESTION.yml:1]: We want questions at https://lemmy.ml/c/lemmy_support, not here. Let's get rid of this entirely and redirect them to lemmy.ml on the `config.yml` file.

File: .github/ISSUE_TEMPLATE/hexbear.yml
Comment (alectrocute) [inline .github/ISSUE_TEMPLATE/hexbear.yml:1]: Irrelevant and unused.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 7]
```diff
@@ -1,66 +1,67 @@
...
-        - Any browser console errors?
+        - Describe your environment (OS, browser, model of smartphone, etc)
+        - If relevant, also share any console errors and/or screenshots here.
     validations:
       required: true
   - type: input
     id: lemmy-ui-version
     attributes:
-      label: Version
-      description: Which Lemmy UI version do you use? Displayed in the footer.
```
Comment (alectrocute) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:57]: Too specific. We may not always display it in the footer in the future, so let's make this more timeless.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml (hunk 1 above)
Comment (SleeplessOne1917) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.yml:1]: The emojis look cool though.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml (hunk 4 above)
Comment (SleeplessOne1917) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.yml:15]: I think "I will post" fits better than "I'd post".

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml [hunk 8]
```diff
@@ -1,54 +1,27 @@
...
     attributes:
       value: |
-        Have a suggestion about Lemmy's UI?
-        For backend issues, use [lemmy](https://github.com/LemmyNet/lemmy/issues/new/choose)
+        Thanks for taking the time to help improve Lemmy-UI by suggesting a feature!
   - type: checkboxes
     attributes:
       label: Requirements
-      description: Before you create a feature request please do the following.
+      description: Before you create a bug report, please carefully check the following –
```
Comment (SleeplessOne1917) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml:12]: Feature request, not bug report.

**Comment by alectrocute** [comment]
@SleeplessOne1917 Thanks for the proofread and good catches! I've made the suggested changes and added the emojis back in.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml (hunk 4 above)
Comment (SleeplessOne1917) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.yml]: Actually, on reflection imperative case seems better, e.g. "please post to..."

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml [hunk 9]
```diff
@@ -1,54 +1,27 @@
...
-        For backend issues, use [lemmy](https://github.com/LemmyNet/lemmy/issues/new/choose)
+        Thanks for taking the time to help improve Lemmy-UI by suggesting a feature!
   - type: checkboxes
     attributes:
       label: Requirements
-      description: Before you create a feature request please do the following.
+      description: Before you create a feature request, please carefully check the following –
       options:
-        - label: Is this a feature request? For questions or discussions use https://lemmy.ml/c/lemmy_support
+        - label: This is a feature request and not a bug report. Otherwise, I will create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml&title=%5BBug%5D%3A+) instead.
```
Comment (dessalines) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml:14]: I'm guessing that Bug doesn't need to be in the title now.

File: .github/ISSUE_TEMPLATE/QUESTION.yml
Comment (dessalines) [inline .github/ISSUE_TEMPLATE/QUESTION.yml:1]: Thanks, we could def do without questions here.

File: .github/ISSUE_TEMPLATE/hexbear.yml
Comment (dessalines) [inline .github/ISSUE_TEMPLATE/hexbear.yml:1]: They haven't really been using these anyway.

**Review by SleeplessOne1917 — APPROVED** [review]

**Pull request merged**